scripts/qa_crowbarsetup.sh

```diff
@@ -4525,6 +4525,10 @@ function onadmin_prepare_crowbar_upgrade
         # move nodes to upgrade mode
         safely crowbar_api_request POST $crowbar_api /installer/upgrade/prepare.json
     else
+        if ! crowbarctl upgrade repocheck crowbar; then
+            complain 11 "Error running 'crowbarctl upgrade repocheck crowbar'"
+        fi
```

Comment:
Or what about `safely crowbarctl upgrade repocheck crowbar  --format plain > some_file` and later just grep that file for "missing" ?